docs/quick.rst

```diff
@@ -191,3 +195,7 @@ implements the dictionary interface::
     True
 
 For more, see :ref:`attributes`.
+
+Save the file to finalize changes::
+
+    >>> f.close()
```

Comment:
Let's leave this out from here, and just describe it in 'creating a file' - it's a bit strange to put it in the attributes section but not e.g. groups, and if we repeat it everywhere then the quickstart isn't so quick.

It's also not really accurate to describe this command as 'sav[ing]' the file - that implies we're constructing something in memory and then writing it out on close, which is not really the case (although close might do some writes as it flushes caches).